docs/source/DeveloperGuide/DevelopmentEnvironment/DeveloperInstallation/editingCode.rst

```diff
@@ -170,6 +169,14 @@ Some useful commands::
 and specifically `miniconda <https://conda.io/en/latest/miniconda.html>`_.
 Use it if you like, if you understand virtualenv you can understand conda environments.
 
+Some useful conda commands::
```

Comment:
I am not too much in favour of adding generic documentation of other software within our own doc. People should know how to look for a tutorial :-)

Reply:
I think a few conda commands here are fine. At the same time, me I have done
```
conda env create
```
which takes the environment.yml by default.

Reply:
yes, at least write teh conda command that actually set up the DIRAC environment